scripts/generate-sw.js

```diff
@@ -0,0 +1,8 @@
+var path = require('path');
+var swPrecache = require('sw-precache');
+var __dirname = 'themes/aframe/source';
+
+swPrecache.write(path.join('scripts','sw.js'), {
```

Comment:
@ngokevin Oh that would be good! Correct. I think it's self contained in the code below (`script/sw.js`)